src/components/AddressSearch/index.js

```diff
@@ -510,15 +539,17 @@ function AddressSearch({
                         styles={{
                             textInputContainer: [styles.flexColumn],
                             listView: [
-                                StyleUtils.getGoogleListViewStyle(styles, displayListViewBorder),
+                                StyleUtils.getGoogleListViewStyle(displayListViewBorder),
```

Comment:
Why this change? It seems wrong to turn a boolean param into a style object.

Reply:
This is not my update. It comes from merging main branch.

Reply:
https://github.com/Expensify/App/pull/32264/commits/226f885ddee6033bbce5ae182f5a74586ee4a5c0#diff-016b8bf57995228d9b24aff3cba5fa9e8a40e493dcc08f7c36af6fac1b4e102c

Here @koko57 added styles to getGoogleListViewStyle function.
Maybe auto-merge deleted styles. I will updated it as before.